### PR TITLE
Add instance name to comments mailer

### PR DESCRIPTION
**Pull request by masaball**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -30,6 +30,7 @@ def create
     @comment.comment = params[:comment][:comment]
 
     if @comment.valid? && (Settings.recaptcha.blank? || verify_recaptcha(model: @comment))
+      @comment.subject = @comment.subject.prepend("#{Settings.name}: ")
```
Comment (cjcolvar) [inline app/controllers/comments_controller.rb:33]: I think there is a cleaner way to prepending the site name without adding it into `@comment.subject` to strip it back out again in the views.  How about doing? `subject: "#{Settings.name}: #{@comment.subject}"` at https://github.com/avalonmediasystem/avalon/blob/90832296809f3684405acb7b4e44d7ed0720d92e/app/mailers/comments_mailer.rb#L20

**Review by cjcolvar — COMMENTED** [review]
Those tests are perfect!  Thanks for filing in this gap in code coverage.

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (masaball) [reply app/controllers/comments_controller.rb:33]: Good call. That is a much better way to handle that. Will make the change.

**Review by cjcolvar — APPROVED** [review]
Looks great!

**Pull request merged**